### PR TITLE
Add new member to README.md

### PR DESCRIPTION
**Pull request by ebicoglu**
added me to the Readme.md

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/wg-maintainers?pullRequest=44) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/wg-maintainers?pullRequest=44) before we can accept your contribution.<br/><br/>:x: ebicoglu [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/wg-maintainers?pullRequest=44)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/wg-maintainers?pullRequest=44) it.</sub>

File: README.md [hunk 1]
```diff
@@ -42,3 +42,4 @@ This committee meets at 21:00 UTC on the second Wednesday of every month. All co
 - Tony Qu - @tonyqus
 - SeanKilleen - Github [@SeanKilleen](https://github.com/SeanKilleen), Twitter [@sjkilleen](https://twitter.com/sjkilleen)
 - Aaron LaBeau - @biozal 
+- Alper EBICOGLU - @ebicoglu
```
Comment (rprouse) [inline README.md:45]: Why ALL CAPS for your last name?

**Review by rprouse — COMMENTED** [review]
You will need to sign the CLA. Silly for this, but required.

File: README.md (hunk 1 above)
Comment (enisn) [reply README.md:45]: See https://github.com/dotnet-foundation/wg-maintainers/pull/47

**Pull request closed without merging**